config/locales/fr.yml

```diff
@@ -230,9 +230,11 @@ fr:
     event_summary: 'Recharge pour %{name}.'
     form:
       strength: Concentration
+      stregth_unit: 'Unité de concentration'
```

Comment:
This should be `strength_unit`. This is causing a test in `./spec/compare_locales_support_spec.rb` to fail.

Reply:
Thanks, fixed this in my latest commit 👍